static/app/components/events/autofix/autofixSteps.tsx

```diff
@@ -456,3 +451,55 @@ const ProgressContainer = styled('div')`
 const ProgressStepContainer = styled('div')`
   grid-column: 1/-1;
 `;
+
+function LogComponent({html}: {html: string}) {
+  const [expanded, setExpanded] = useState(false);
+  const [isExpandable, setIsExpandable] = useState(false);
+  const logRef = useRef(null);
+
+  useEffect(() => {
+    const checkExpandable = () => {
+      if (logRef.current) {
+        const {scrollHeight, clientHeight} = logRef.current;
+        setIsExpandable(scrollHeight > clientHeight + 16);
+      }
+    };
+
+    checkExpandable();
+    window.addEventListener('resize', checkExpandable);
+    return () => window.removeEventListener('resize', checkExpandable);
+  }, [html]);
+
+  const toggleExpand = () => {
+    if (isExpandable) {
+      setExpanded(!expanded);
```

Comment:
nit: better to do `setExpanded(oldState => !oldState)` - react prefers this so you can't use a stale state value

Reply:
Done in next commit